### ISSUE TITLE
How to use MS:1001239 (frag: immonium ion) to annotate immonium ions on specific amino acids?

### ISSUE THREAD

**Issue by hbarsnes** [HUPO-PSI/mzIdentML#9]
What is the intended use of MS:1001239 (frag: immonium ion) and how can this be used to annotate immonium ions on specific amino acids?

Here's what we currently do in the PeptideShaker mzIdentML export using the outdated PRIDE CV terms:

```
<IonType charge="1" index="4 5">
   <FragmentArray measure_ref="Measure_MZ" values="120.0808411"/>
   <FragmentArray measure_ref="Measure_Int" values="14483.125"/>
   <FragmentArray measure_ref="Measure_Error" values="6.53397579810644E-5"/>
   <cvParam cvRef="PRIDE" accession="PRIDE:0000244" name="immonium F"/>
</IonType>
```

How would the same be done with MS:1001239 (frag: immonium ion)?

**Comment by andrewrobertjones** [comment]
Documentation in the XSD explains this: 
For immonium ions, the index is the position of the identified ion within the peptide sequence - if the peptide contains the same amino acid in multiple positions that cannot be distinguished, all positions should be given.

**Comment by hbarsnes** [comment]
Thanks Andy. Now implemented in the updated PeptideShaker mzid 1.2 example file. :)
